README.md

```diff
@@ -1 +1,29 @@
-This repository is used to keep hr-activities code
+## Scope
+The Employee Management System is designed to manage the employee data 
+with an organisation. t allows users to perform various CRUD (Create, Read, Update, Delete) operations 
+on employee records. The system provides functionalities such as adding new employees, updating existing records, deleting employees, 
+and retrieving employee information.
+
+## Technologies Used :
+* Java
+* Maven
+* Spring Boot
+
+## Prerequisites :
+Before running the application,<br>
+Please ensure that you have the following prerequisites installed on your system:
+
+* Java Development Kit (JDK)
+* IntelliJ Idea or Spring Tools Suite
+* Post GreSQL
+
+## Building the Application :
+* Clone into the GitHub Repostitory : git clone https://github.com/sutirthak/hr-activities.git
+* Open the Maven Project in the IDE
+* Create a Employee Database in PostGreSQL
+* Update the application.properties file with Database Credentials
+* Run Application.java
```

Comment:
Maven does not depend on IDE. 
So give the command how it can be compiled and run like:
mvn clean install => to build the jar file
Then java -jar <executable jar file name> to run the jar

Reply:
Updated the documentation and running sucessfuly.